client/signup/steps/design-picker/index.jsx

```diff
@@ -26,6 +36,14 @@ class DesignPickerStep extends Component {
 		showOnlyThemes: false,
 	};
 
+	componentDidMount() {
+		this.fetchThemes();
```

Comment:
Should we check if `this.props.showOnlyThemes === true`?

Reply:
🆗 yeah we should. I think we can get rid of `showOnlyThemes` actually, it's only there so we don't break another flow, but that flow is no longer needed and can be removed. That's another PR though.